### ISSUE TITLE
Could we clarify the examples in README?

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/PSSMGen#5]
Hi Cunliang, I was wondering whether you have time to clarify a couple of things for the examples in REAMDE?

1.   Example 1: “Get consistent PSSM and PDB files for a specific protein-protein complex”. The first thing my students asked was "Why do we need pdb files to calculate pssms"? Could you please provide an example using fasta files? I saw it at the bottom of the readme in two separate examples. But it is obviously not clear for my students and probably won’t be clear for the users.

2. Example: “Using customed PSSM files”, we created an issue #4 as it caused error messages

In summary, could we clarify the following things in the README:

1. pssmgen can calculate pssm from fasta files and can map the pssm files to the pdb files so that we can check the conservation patterns on the pdb

2.  Provide one example at the beginning of the README (not the bottom) that can do the following two steps: 1) use fasta files as input and calculate psssm; 2) map pssm files to pdb files (edited) 

3. The existing examples in README are good. We can keep them.

**Comment by CunliangGeng** [comment]
The README has been updated accordingly. Close the issue now.
